packages/logger/src/trace-effect.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  createTraceEffect as createTraceEffectBase,
+  TraceEffectCallbacks,
+  TraceEffectOptions,
+} from "@saleor/apps-otel/trace-effect";
+
+import { rootLogger } from "./root-logger";
```

Comment:
this is wrong. App should import both logger and otel. App is creating its own logger with own settings. Here you break the rules, packages talk to each other directly

IF you wan to add this "default" behavior, you can copy-paste this file to each app